apps/nextjs-website/src/lib/oneTrust.ts

```diff
@@ -0,0 +1,22 @@
+export const ExtractNoticeIDFromOneTrustURL = (
```

Comment:
**suggestion**: What about lowercase it? If you do so you should change the references
```suggestion
export const extractNoticeIDFromOneTrustURL = (
```